src/components/admin/feedback/ManageFeedback.jsx

```
@@ -51,6 +52,7 @@ export default function ViewFeedback() {
     // Delete feedback by ID
     const handleDeleteFeedback = async (feedbackId) => {
         try {
+            const data = await api.del(`customer-feedbacks/${feedbackId}`)
             const data = await api.del(`customer-feedbacks/${feedbackId}`)
```

Biome (1.9.4)

[error] Shouldn't redeclare 'data'. Consider to delete it or rename it.

'data' is defined here:


(lint/suspicious/noRedeclare)